.github/workflows/publish.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           credentials: "${{ secrets.CHARMHUB_TOKEN }}"
           github-token: "${{ secrets.GITHUB_TOKEN }}"
           channel: "${{ steps.channel.outputs.name }}"
+          upload-image: "false"
```

Comment:
Keeping up with yesterday's discussion about security, I think we should upload the image at least every time there's a change to the project

Reply:
Closed.